node/alphanet/src/lib.rs

```diff
@@ -147,6 +147,7 @@ mod tests {
     /// Creates two nodes, one boot node and secondary node booting from it.
     /// Waits until they produce block with transfer money tx.
     #[test]
+    #[ignore]
```

Comment:
Why this needs to be commented out?

Reply:
this test now hits the assert that we don't send Reset twice with the same block index

Reply:
Yes, I realized now that this test it just broken, see https://github.com/nearprotocol/nearcore/pull/646